spec/views/staff/shared/message_spec.rb

```diff
@@ -9,11 +9,16 @@
 
   context "when passed a flash message with a hash" do
     it "renders a flash message with a hash that has title and errors keys" do
```

Comment:
Could you update the spec description as well, please, since this seems to be checking for the error messages and not the error keys?

Reply:
Oooh, good catch